examples/gsutil/main.tf

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  version = "~> 1.20"
+  project = "${var.project_id}"
+  region  = "${var.region}"
+  zone    = "${var.region}-a"
+}
+
+module "startup-scripts" {
+  source                       = "../../"
+  enable_init_gsutil_crcmod_el = true
+  enable_get_from_bucket       = true
+}
+
+data "google_compute_image" "os" {
+  project = "centos-cloud"
+  family  = "centos-7"
+}
+
+resource "random_id" "resource_name_suffix" {
+  byte_length = 4
+}
+
+#    webhook_token = "${random_id.startup_scripts_github_webhook_token.hex}"
+
+# Storage bucket used by startup-script-custom and stdlib::get_from_bucket
+resource "google_storage_bucket" "example" {
+  name          = "startup-scripts-${random_id.resource_name_suffix.hex}"
+  storage_class = "REGIONAL"
+}
+
+resource "google_storage_bucket_object" "message" {
```

Comment:
Left off here.  When running kitchen converge from inside `make integration_test_shell` and with `source test/ci_integration.sh && setup_environment`

```
       Error: Error applying plan:

       1 error(s) occurred:

       * module.example.google_storage_bucket.example: 1 error(s) occurred:

       * google_storage_bucket.example: googleapi: Error 403: ci-startup-scripts@startup-scripts.iam.gserviceaccount.com does not have storage.buckets.create access to project 1010336466588., forbidden

       Terraform does not automatically rollback in the face of errors.
       Instead, your Terraform state file has been partially updated with
       any resources that successfully completed. Please address the error
       above and apply again to incrementally change your infrastructure.


>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Converge failed on instance <gsutil-local>.  Please see .kitchen/logs/gsutil-local.log for more details
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration
```